### PR TITLE
added Language property to ParseOption and CompilationOption

### PR DESCRIPTION
**Pull request by heejaechang**
property name followed the compiler code base pattern.

### CODE REVIEW

**Comment by heejaechang** [comment]
@gafter @Pilchie @CyrusNajmabadi added new public API to compilation/parse option.

this is for master not rc.

**Comment by CyrusNajmabadi** [comment]
👍

File: src/Compilers/Core/Portable/Compilation/CompilationOptions.cs [hunk 1]
```diff
@@ -339,6 +339,11 @@ internal bool CanReuseCompilationReferenceManager(CompilationOptions other)
                 && object.Equals(this.AssemblyIdentityComparer, other.AssemblyIdentityComparer);
         }
 
+        /// <summary>
+        /// Gets the source language ("C#" or "Visual Basic").
+        /// </summary>
+        public abstract string Language { get; }
```
Comment (gafter) [inline src/Compilers/Core/Portable/Compilation/CompilationOptions.cs]: I think I'd prefer `LanguageName`.

File: src/Compilers/Core/Portable/Compilation/ParseOptions.cs [hunk 2]
```diff
@@ -31,6 +31,11 @@ internal ParseOptions(SourceCodeKind kind, DocumentationMode documentationMode)
         }
 
         /// <summary>
+        /// Gets the source language ("C#" or "Visual Basic").
+        /// </summary>
+        public abstract string Language { get; }
```
Comment (gafter) [inline src/Compilers/Core/Portable/Compilation/ParseOptions.cs]: Same Here (I think I'd prefer `LanguageName`)

**Comment by gafter** [comment]
@mattwar @AnthonyDGreen @jaredpar Do you care to chime in on this?

File: src/Compilers/Core/Portable/Compilation/CompilationOptions.cs (hunk 1 above)
Comment (CyrusNajmabadi) [inline src/Compilers/Core/Portable/Compilation/CompilationOptions.cs]: We have SyntaxNode.Language, ISymbol.Language and Project.Language.  I don't believe we really use 'LanguageName' in our public APIs.  I think this should probably match those other APIs.  Wht do you think Neal?

**Comment by heejaechang** [comment]
retest  windows_debug_unit64_prtest please

**Comment by heejaechang** [comment]
d15 overwriting d14 issue

**Comment by AnthonyDGreen** [comment]
This looks entirely consistent with the other Language properties we have. Are we aware of any other types that we should add this property to?

**Pull request merged**